_data/accessibility-tests/checkbox.yml

```
@@ -0,0 +1,54 @@
+title: Checkbox
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain checkbox requirements. 
+    summary_additional: The checkbox has a visible label above it to indicate its purpose. For example, "Select your state."
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons in a checkbox, all labels have short descriptions that are easy to understand and act on. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.6
+# Keyboard navigation tests
+  - summary: Focus indicator on checkbox is clearly visible.
+    summary_additional: When you use a keyboard to focus on a checkbox group, you will see a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.1
+  - summary: Checkbox focus follows logical content order on desktop.
```

@finekatie @amycole501 @alex-hull 

Can we remove "on desktop" from this summary? It seems like it would be applicable at all screen widths.

```suggestion
  - summary: Checkbox focus follows logical content order.
```

---

Yes, I think it is fine to remove that.

---

Resolved in [5808ba8](https://github.com/uswds/uswds-site/pull/2803/commits/5808ba81a1e73b90ad5129c8a440f7a27c6ad4a9)